nrfx/drivers/include/nrfx_comp.h

```diff
@@ -66,15 +66,23 @@ extern "C" {
  */
 typedef void (* nrfx_comp_event_handler_t)(nrf_comp_event_t event);
 
-/** @brief COMP shortcut masks. */
+/**
+ * @brief COMP shortcut masks.
+ * 
+ * @note This enum is deprecated. Use @ref nrf_comp_short_mask_t instead.
```

Comment:
nit, there is doxygen command for this:
```suggestion
 * @deprecated Use @ref nrf_comp_short_mask_t instead.
```